### PR TITLE
Get tests passing

### PR DESCRIPTION
**Pull request by tdenniston**

### CODE REVIEW

File: test/src/unit-capi-serialization-dense_array.cc [hunk 1]
```diff
@@ -3013,120 +3013,6 @@ TEST_CASE_METHOD(
   remove_temp_dir(temp_dir);
 }
 
-TEST_CASE_METHOD(
```
Comment (Shelnutt2) [inline test/src/unit-capi-serialization-dense_array.cc:3016]: Why remove this? Is it not passing?

Comment (tdenniston) [reply test/src/unit-capi-serialization-dense_array.cc:3016]: The test included the following scenario:

- Open an array
- Allocate a query on that array
- Close the array
- Open the array again
- Serialize the query

But the query object internally hangs on to a pointer to the `ArraySchema` from the array it was allocated with, and that object is deleted when the array is closed the first time. This test exists in upstream also, but we should probably remove it from there as well (this is sort of a "won't fix" scenario, it violates a lot of our assumptions about object lifetimes).

Comment (Shelnutt2) [reply test/src/unit-capi-serialization-dense_array.cc:3016]: okay makes sense thanks for clarifying.

**Review by Shelnutt2 — APPROVED** [review]
lgtm

File: .travis.yml [hunk 2]
```diff
@@ -39,7 +39,7 @@ install:
       fi
 
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-        brew uninstall --ignore-dependencies libjpeg libtiff;
+        brew uninstall --ignore-dependencies gdal giflib libjpeg libtiff;
```
Comment (Shelnutt2) [inline .travis.yml]: Why are we getting these errors now and not in core tiledb? Is travis here using different versions of osx/ubuntu?

Comment (tdenniston) [reply .travis.yml]: I don't really know, currently I'm assuming it's some bad cached state on the worker image (hopefully if I can fix it I can then remove these explicit brew steps from the travis file).

**Review by Shelnutt2 — APPROVED** [review]
lgtm

**Pull request merged**